src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -431,6 +438,44 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 			}
 		}));
 
+		// Register DragAndDrop support
+		this._register(new DragAndDropObserver(this.root, {
```

Comment:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/10746682/78562867-58a65500-781a-11ea-91db-0b75a0c46434.png">

Shall we have drop overlay only on extensions views excluding the search box ?

Reply:
Based on my conversation with @joaomoreno, we wanted to keep things simple and that is why we are set up the drop target/drop overlay to be the shown on the whole extensions view. If you feel strongly about this, or there is a prior pattern not to show the drop overlay over the search box, I am happy to update it.